### PR TITLE
feat: Render HTML based content

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary
- Render HTML content on User statement and discussion
- Display first post in a topic (previously hidden)
- Imporve twitter card
- image

Closes: https://github.com/snapshot-labs/workflow/issues/189

### How to test

1. Go to a discussion page. http://localhost:8080/#/s:lido-snapshot.eth/proposal/0xa478fa5518769096eda2b7403a1d4104ca47de3102e8a9abab8640ef1b50650c/discussion
2. Should be formated similar to https://research.lido.fi/t/organize-the-lido-alliance-program-as-a-lido-dao-adjacent-borg/8173/3

### To-Do

- [x] Find if some case is missing

### CODE REVIEW

File: apps/ui/src/views/Proposal/Discussion.vue [hunk 1]
```diff
@@ -21,6 +22,9 @@ onMounted(async () => {
     console.error(e);
   }
 });
+const toMarkdown = computed(() =>
+  turndownService({ discussion: discussion.value || '' })
+);
```
Comment (wa0x6e) [inline apps/ui/src/views/Proposal/Discussion.vue:27]: computed const should be together, before onMounted

File: apps/ui/src/views/SpaceUser/Statement.vue [hunk 2]
```diff
@@ -57,6 +58,8 @@ watch(userId, loadStatement, { immediate: true });
 watchEffect(() =>
   setTitle(`${props.user.name || userId.value} ${props.space.name}'s profile`)
 );
+
+const toMarkdown = computed(() => turndownService());
```
Comment (wa0x6e) [inline apps/ui/src/views/SpaceUser/Statement.vue:62]: Same here, computed const should be grouped together

**Comment by wa0x6e** [comment]
Incorrect escape on `[` and `]` char on https://bafybeib554rbng74f3apb2angnydsm4xmzl4aqnspgqtwjpnjwefnzozo4.on.fleek.co/#/s:aave.eth/proposal/0xf87cf0761b27becf6c8d18bbb457c9e6bf6b7aa436cdb0d197ad2d93a495ed04/discussion

**Comment by wa0x6e** [comment]
On http://localhost:8080/#/s:aave.eth/proposal/0x9e2e3bd26866212d0cbd8e7cefcfa21eec9522202fd25b02456b8ff59371db08/discussion

Rendering of the table is missing, and the post is cut (missing everything after the table)

**Comment by wa0x6e** [comment]
On http://localhost:8080/#/s:apecoin.eth/proposal/0xaa3b8b16b4e6193e8c0feb8d9fd0fed9873659bc27ffa4ec9f5792f323f8c766/discussion

Another escape issue, search for `\-@Facilitators`

**Comment by wa0x6e** [comment]
http://localhost:8080/#/s:apecoin.eth/proposal/0xff1829f8e2fb778946ef83050858a54582da50da3de3acb67c74f5e38c1da0aa/discussion

Issue with bullet point, always giving `1\.` instead of `1.`

**Comment by wa0x6e** [comment]
`ul` and `ol` does not have a padding bottom of 8px, compared to all other tags, making post ending by these tags closer to the footer than the other post

Screenshot 2024-09-02 at 23 17 33

**Comment by ChaituVR** [comment]
> On http://localhost:8080/#/s:safe.eth/proposal/0x2c2ad686e94fd97903de8ee520b90d400c98a1985e31b21416da095b28373f18/discussion
> 
> Some `_` are escaped, search for `@amy\_safe`

It is just here discord id:

image

image

**Comment by wa0x6e** [comment]
> `ul` and `ol` does not have a padding bottom of 8px, compared to all other tags, making post ending by these tags closer to the footer than the other post
> 
> Screenshot 2024-09-02 at 23 17 33

Same issue with table missing bottom padding:

Screenshot 2024-09-03 at 12 26 35

**Comment by wa0x6e** [comment]
In http://localhost:8080/#/s:apecoin.eth/proposal/0xaa3b8b16b4e6193e8c0feb8d9fd0fed9873659bc27ffa4ec9f5792f323f8c766/discussion

In the penultimate post, the "-@Facilitators" is linking to the same host (localhost:8080)

**Comment by bonustrack** [comment]
Can we merge this?

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**